core/crypto/crypto_store/crypto_suites.hpp

```diff
@@ -81,13 +81,20 @@ namespace kagome::crypto {
     ~EcdsaSuite() override = default;
 
     EcdsaKeypair generateRandomKeypair() const noexcept override {
-      return ecdsa_provider_->generate().value();
+      auto kp = ecdsa_provider_->generate();
+      if (kp) {
+        return kp.value();
+      }
+      return EcdsaKeypair{};
```

Comment:
I dislike discarding the error here. If it is truly well-justified, please leave a comment.

Reply:
What's the alternative in case of noexcept?